src/Transport/Failover/FailoverTransport.cs

```diff
@@ -1,1689 +1,1701 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I don't know what you have done here but can you make it so just the lines of code you need changed are changed not the whole file, as its impossible to diff like this

Reply:
I see what you mean... not sure why that happened on this one and not the other files... I just pulled from upstream master and resolved the conflict using the conflict editor - did not even require any hand editing. (I think it would have been cleaner with a rebase - but I don't think I can do that in an existing public branch.)

Reply:
You can make your own branch and rebase / sort this nothing stopping you doing that and sending a new pr and still keeping the original authors commit.

Reply:
Tbh the original commit message also should to be corrected to have the jira prefix at the front. So youre going to want to git commit amend that anyhow